Formula/stubby.rb

```diff
@@ -0,0 +1,56 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver service based on getdns"
+  homepage "https://getdnsapi.net/blog/dns-privacy-daemon-stubby/"
+  url "https://github.com/getdnsapi/stubby/archive/v0.1.0.tar.gz"
+  sha256 "6210291850d6f7f124a5ba4bcee2f50814f020b7a0c4e67c6646bfe35ed5dd5b"
+  head "https://github.com/getdnsapi/stubby.git", :branch => "master"
+
+  link_overwrite "bin/stubby"
```

Comment:
You are right, I thought it worked on my first test but just retried and it failed. Will get a 1.1.3 release out.